scripts/batch-score.coffee

```diff
@@ -0,0 +1,107 @@
+# Description:
+#   Script for batch wise score.
+#
+# Configuration:
+#   INFO_SPREADSHEET_URL
+#
+# Commands:
+#   hubot score bxx
+# 
+# Author:
+#   aseem09
+
+module.exports = (robot) ->
+
+  # using this, stores the data in brain's "scorefield" key
+  scorefield = () ->
+    Field = robot.brain.get("scorefield") or {}
+    robot.brain.set("scorefield",Field)
+    Field
+  
+  # response for score status of any <batch>
+  stringLength = (str) ->
+    string=String(str)
+    temp=string.split('')
+    return temp.length
```

Comment:
Can be simplified to `String(str).split('').length`. Many web dev languages use `strlen` to calculate the length of a string. We may want to change this to `strlen` if coffee already does not have any method to achieve so.